tests/core/engine_adapter/test_integration.py

```diff
@@ -45,6 +45,7 @@ def __init__(
         test_type: str,
         engine_adapter: EngineAdapter,
         gateway: str,
+        is_cloud_database: bool = False,
```

Comment:
The parameters are already marked as being `remote` or `local`. Could `remote` mark be used instead of adding this?

Reply:
This was actually powered by the `remote` mark already. It was more convienient to make it a property of the `TestContext` vs injecting the `request` fixture everywhere it was needed and then calling `request.node.get_closest_marker("remote") is not None` to figure out if the `remote` mark was set.

However, i've renamed the property on `TestContext` from `is_cloud_database` to `is_remote`, is that better?